src/Workspaces/Core/Portable/Options/GlobalOptionService.cs

```diff
@@ -0,0 +1,198 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Composition;
+using Microsoft.CodeAnalysis.Options.Providers;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.Options
+{
+    [Export(typeof(IGlobalOptionService)), Shared]
+    internal class GlobalOptionService : IGlobalOptionService
```

Comment:
This is the old OptionService code.  It didn't change.  Unfortunately, git doesn't seem to recognize it as a file rename.